src/drive/web/modules/upload/index.js

```diff
@@ -204,7 +204,7 @@ const uploadFile = async (client, file, dirID) => {
 export const getFileFullpath = async (client, file, dirID) => {
   const resp = await client.collection('io.cozy.files').get(dirID)
   const parentDirectory = resp.data
-  return `${parentDirectory.path}/${file.name}`
+  return `${parentDirectory.path}/${file.name}`.replace('//', '/')
```

Comment:
We can't have a filename with `//` in it?

Reply:
I get an error from the stack when I try to use `/` in a filename (rename). Also, I can't use `/` in a filename on linux so I can't test an upload with `/` in filename